build.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: "YouTube Metadata"
```

Comment:
Thanks for the contribution!
What exactly is this file for?

Reply:
`build.yaml` is used in https://github.com/oddstr13/jellyfin-plugin-repository-manager/ for some ez automation

Reply:
ahh nice, I believe that was what dkanada was talking about, I had yet to start that.

Reply:
This file is used in our build infrastructure, and maybe most importantly also used by my new tool, [jprm](https://github.com/oddstr13/jellyfin-plugin-repository-manager).

Reply:
It is mostly just metadata needed by the plugin repository manifest